lib/insert/insert_donation.rb

```diff
@@ -173,7 +173,7 @@ def self.insert_payment(kind, fee_total, donation_id, data)
   def self.insert_donation(data, entities)
     d = Donation.new
     d.date = data['date']
-    d.anonymous = data['anonymous']
+    d.anonymous = data['anonymous'].nil? ? false : true
```

Comment:
We should move this into a `:before_save` on Donation too.

Reply:
It's already there, I was debugging and committed this by accident, sorry!